transcription/transcription.go

```diff
@@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
+func ConvertAudioIntoRequiredFormat(fn string) error {
```

Comment:
Thanks for thorough explanation! Followed that template, and getting this error (also tried a bunch of other variations). Ideas?

<img width="487" alt="screen shot 2016-03-13 at 7 20 50 pm" src="https://cloud.githubusercontent.com/assets/4250521/13732099/cc1892f8-e950-11e5-8312-be005bd66de6.png">